README.md

```diff
@@ -35,16 +35,18 @@ See more scripts in the [package.json](package.json) file or the [makefile](Make
 
 ### Connection options
 
-| Option          |  Description                                    |                     |
-|-----------------|-------------------------------------------------|---------------------|
-| accessKeyString | Hex private part of an access key               | string              |
-| debug           | Whether to turn debugging on                    | boolean             |
-| signRequest     | Whether to sign all requests sent to the server | boolean             |
-| transportType   | Transport type to use for server requests       | "http" or "graphql" |
-| xPriv           | bsv HDPrivateKey instance                       |                     |
-| xPrivString     | HD Private key string                           | "xprv....."         |
-| xPub            | bsv HD Public key instance                      |                     |
-| xPubString      | HD Public key string                            | "xpub....."         |
+| Option          | Description                                     |             |
+|-----------------|-------------------------------------------------|-------------|
+| accessKeyString | Hex private part of an access key               | string      |
+| accessKey       | bsv PrivateKey instance                         |             |
+| adminKey        | Admin Key string                                | "xprv....." |
+| debug           | Whether to turn debugging on                    | boolean     |
+| signRequest     | Whether to sign all requests sent to the server | boolean     |
+| xPriv           | bsv HDPrivateKey instance                       |             |
+| xPrivString     | HD Private key string                           | "xprv....." |
+| xPub            | bsv HD Public key instance                      |             |
+| xPubString      | HD Public key string                            | "xpub....." |
+| xPubID          | XPub Id string                                  | string      |
```

Comment:
Are you sure we are able to make a request with xPubID only

Reply:
Updated